.env.template

```diff
@@ -10,3 +10,4 @@ JWT_REFRESH_TOKEN_EXP_IN_SEC=7200
 JWT_PUBLIC_KEY_BASE64=
 JWT_PRIVATE_KEY_BASE64=
 DEFAULT_ADMIN_USER_PASSWORD=admin-example
+MYSQL_PLATFORM=
```

Comment:
Does it work on other systems if the value is an empty string? Otherwise we should have a default value here.

Reply:
I believe we have tested it on mac and ubuntu, not sure about windows